cygnus-ngsi/README.md

```diff
@@ -50,7 +50,7 @@ Simply configure the FIWARE repository if not yet configured:
     $ cat > /etc/yum.repos.d/fiware.repo <<EOL
     [Fiware]
     name=FIWARE repository
-    baseurl=http://repositories.testbed.fi-ware.eu/repo/rpm/x86_64/
+    baseurl=http://repositories.testbed.fiware.eu/repo/rpm/x86_64/
```

Comment:
This is the same than above. Anyway, fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1375/commits/b0b2e859498b6c1a20112df2a0b373f9b41dade4